nmr_commons/tracking/Trajectory.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import csv
+import copy
+
+
+class Trajectory:
+    """Trajectory class where trajectory is a list of point and each point is a tuple of two points in space"""
+    ####
+    # Constructor
```

Comment:
comment in two lines above doesn't contain any useful information, it's better to remove it